server.js

```diff
@@ -28,13 +29,21 @@ const app = express()
 // Set up error tracking with Sentry
 const SENTRY_URL = process.env.SENTRY_URL
 const Raven = require('raven')
+const redis = require('./lib/redis')
+const _ = require('lodash')
 
 Raven.config(SENTRY_URL).install()
 
 if (process.env.NODE_ENV !== 'test') {
-  app.use(expressMetrics({
-    port: 8091,
-  }))
+  app.use(metricsMiddleware())
+  app.get('/metrics', (req, res) => {
+    redis.keys(`${metricsPrefix}*`, function(error, keys) {
```

Comment:
The `keys` command is very heavy. We should use something like a set to store the accessed urls.

> It may ruin performance when it is executed against large databases. This command is intended for debugging and special operations, such as changing your keyspace layout. Don't use KEYS in your regular application code.

https://redis.io/commands/keys

Reply:
Yup. I wasn't sure on how to achieve the same results where I could query for a set of results. I'll check out set.